src/histolab/tiler.py

```diff
@@ -67,6 +68,37 @@ def box_mask(self, slide: Slide) -> np.ndarray:
     def extract(self, slide: Slide):
         raise NotImplementedError
 
+    def locate_tiles(
+        self, slide: Slide, alpha: int = 128, outline: str = "red"
+    ) -> Image:
+        """Place tiles references on the slide thumbnail image
+
+        Parameters
+        ----------
+        slide : Slide
+            Slide reference where placing the tiles
+        alpha: int
+            The alpha level to be applied to the slide thumbnail, default to 128.
+        outline: str
+            The outline color for the tile annotations, default to 'red'.
+        Returns
+        -------
+        img
+            PIL Image of the slide thumbnail with the extracted tiles outlined
+        """
+        if not os.path.exists(slide.thumbnail_path):
+            slide.save_thumbnail()
+        tiles_coords = (tc[1] for tc in self._tiles_generator(slide))
+        img = Image.open(slide.thumbnail_path)
+        img.putalpha(alpha)
+        draw = ImageDraw.Draw(img)
+        for coords in tiles_coords:
+            rescaled = np.array(tuple(coords)).reshape(2, 2) / np.power(
+                10, np.ceil(np.log10(slide.dimensions)) - 3
+            ).astype(int)
```

Comment:
to do this I suggest you use `utils.scale_coordinates()` https://github.com/histolab/histolab/blob/master/src/histolab/util.py#L156

although I may be not getting what you're doing with the `log` and `-3` 🤔

Reply:
the same we do here
```python 
    @lazyproperty
    def _thumbnail_size(self) -> Tuple:
        return tuple(
            [
                int(s / np.power(10, math.ceil(math.log10(s)) - 3))
                for s in self.dimensions
            ]
        )
```

Reply:
mmmmmmmm here (lines 96-98) you want to scale the tile coordinates from the dimensions at level 0 to the dimensions of the image you are working on, right? If yes, we can use that function from utils

Reply:
ok i'll do that

Reply:
honestly, I don't remember how we came up with that formula for `_thumbnail_size` 🤨😝

Reply:
@alessiamarcolini can you give me some hint on how to use that func? just to understand the params to pass there :D

Reply:
should be like this
```python
rescaled = scale_coordinates(coords, slide.dimensions, img.size)
```

Reply:
thx